### ISSUE TITLE
Linked documents example response is wrong

### ISSUE THREAD

**Issue by jo** [pouchdb/pouchdb#3800]
In https://github.com/pouchdb/pouchdb/blob/master/docs/_includes/api/query_database.html#L258 the response is wrong. Instead of
```json
            "value": {
                "_id": "album_hunkydory",
                "albumYear": 1971
            }
```
it should read
```json
            "value": {
                "_id": "bowie",
                "albumYear": 1971
            }
```

**Comment by jo** [comment]
Pull is #3801

**Comment by nolanlawson** [comment]
fixed by 2d170d0e567c725d2a426eef44716d41449c64b7
